README.md

```diff
@@ -328,6 +328,7 @@ This repository also provides one such platforms where contributers come over an
 | [Brick Buster Game](https://github.com/kunjgit/GameZone/tree/main/Games/Brick Buster) |
 | [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy) 
 |[Penguins Can't Fly](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Penguins_Can't_Fly)|
+|[Mamba_Mayhem](https://github.com/kunjgit/GameZone/tree/main/Games/Mamba_Mayhem)|
```

Comment:
Place your game 10 places above  as it is giving merge conflicts!!

Reply:
@ishita-43 Sorry for any frustrations caused. I'm learning how to resolve merge conflicts. Verify the changes